README.md

```diff
@@ -3,16 +3,29 @@
 [![Build Status](https://travis-ci.com/vanvalenlab/caliban-toolbox.svg?branch=master)](https://travis-ci.com/vanvalenlab/caliban-toolbox)
 [![Coverage Status](https://coveralls.io/repos/github/vanvalenlab/caliban-toolbox/badge.svg?branch=master)](https://coveralls.io/github/vanvalenlab/caliban-toolbox?branch=master)
 
-DeepCell Toolbox is a collection of data engineering tools for processing, annotating, and packaging optical microscopy images. The framework enables crowdsourced annotations and creates training data for [DeepCell](https://github.com/vanvalenlab/deepcell-tf).
+Caliban Toolbox is a collection of data engineering tools to process and curate crowdsourced image annotations using [Caliban](https://github.com/vanvalenlab/caliban), our data annotation tool. The Toolbox and Caliban work together to generate  annotations for training [DeepCell](https://github.com/vanvalenlab/deepcell-tf).
 
 The process is as follows:
-![flow](./docs/flowchart.png)
 
-Read the documentation at
+1. Raw data is imported using the data loader, which allows the user to select data based on imaging platform, cell type, and marker of interest. 
 
-## Getting Started
+2. The raw data can then be run through deepcell-tf to produce predicted labels
+
+3. After making predictions with deepcell, the raw data is processed to make it easier for annotators to view. This includes adding filters, adjusting the contrast, etc. In addition, multiple channels can be summed together. Following these modifications, the user selects which of these channels will be included for the annotators to see.
+
+4. The size of the images is then modified to make annotation easier. In order to get high quality annotations, it is important that the images are not so large that the annotators miss errors. Therefore, the images can be cropped into overlapping 2D regions to break up large FOVs. For stacks of images, the stack can be sliced into smaller more manageable pieces. 
+
+5. Once the image dimensions have been set, each unique crop or slice is saved as an NPZ file. During this process, a JSON file is created which stores the necessary data to reconstruct the original image after annotation.
 
-DeepCell Data Engineering uses `nvidia-docker` and `tensorflow` to enable GPU processing.  
+6. The NPZ files are then uploaded to a cloud bucket, where they can be accesssed by Figure8. During the upload process, the user specifies an existing job to use a template, which populates the instructions for the annotators and the job settings. During the upload process, a log file is created with the necessary information to download the annotations once the job is completed
+
+7. Once the job is completed, the corrected annotations are downloaded from the AWS bucket, where they are stored as the job progresses.
```

Comment:
AWS bucket vs cloud bucket? Perhaps AWS bucket since we don't support other cloud storage solutions for caliban files at the moment (would change step 6 description)